docs/how-to/uc-boot-parameters.rst

```diff
@@ -0,0 +1,70 @@
+How to Configure Boot parameters on Ubuntu Core 
+================================================
+
+Different from classic versions of Ubuntu (Desktop and Server), `Ubuntu Core`_ is a minimal, secure and strictly
+confined operating system designed to run on embedded systems, such as IoT devices.
+Ubuntu Core is a snap-based operating system, meaning it uses snap packages to manage applications and services.
+Its read-only nature makes it a very secure and reliable operating system but also makes it a bit more difficult to
+configure certain aspects of the system, such as boot parameters.
+
+
+Configure boot parameters
+-------------------------
+
+As UC doesn't have the file :code:`/etc/default/grub` mapped for the kernel parameters, neither the `update-grub`_
+utility, it's necessary to use the `snap set`_. There are two ways to set the kernel parameters:
+
+- `system system.kernel.cmdline-append`_
+- `system system.kernel.dangerous-cmdline-append`_
+
+The first one is used to dynamically append permitted kernel boot parameters that are verified against an `allow list`_
+in the `gadget snap`_, while the second one is used to append kernel boot parameters considered risky according to that
+specific gadget snap. In the case that you're interested in using the :code:`pc-gadget` or the :code:`pi-gadget`, you
+should use the `system system.kernel.dangerous-cmdline-append` option, since the allow list isn't present on those 
+gadget snaps, but if you want to `build a gadget snap`_ yourself, you can implement it.
```

Comment:
```suggestion
should set the kernel parameters via the `system.kernel.dangerous-cmdline-append` option.
This is because those reference gadgets are generic and do not bundle an allowed parameter list.

When creating `a custom gadget snap`_ for a real-time Ubuntu Core, it is advisable to define the list of kernel parameters that are relevant to the use cases.
```